Sources/Model/TransferMethod/HyperwalletPrepaidCardQueryParam.swift

```diff
@@ -20,14 +20,26 @@ import Foundation
 
 /// Representation of the prepaid card query parameters.
 public class HyperwalletPrepaidCardQueryParam: HyperwalletTransferMethodQueryParam {
-    private enum QueryParam: String {
-        case type
+    /// Representation of the transfer method status
+    public enum QueryStatus: String {
+        /// Filter by activated transfer methods
+        case activated = "ACTIVATED"
+        /// Filter by deActivated transfer methods
+        case deActivated = "DE_ACTIVATED"
+        /// Filter only lostOrStolen prepaid cards
```

Comment:
minor : since this documentation, please use separate words instead of joined - `lost or stolen`

Reply:
Done